src/main/java/uk/ac/bath/se/DungeonController.java

```diff
@@ -238,6 +272,17 @@ private void addRoomBounds(Dungeon dungeon, int roomQuantity) {
         }
     }
 
+    private boolean roomCollision(int startHeight, int startWidth) {
+        for (int i = startHeight-1; i <= startHeight+5; i ++) {
```

Comment:
What is 5

Reply:
5 is the room height/width (loop is searching one layer around the outside of the room to allow room for passages)

Reply:
Declare it as a variable with an easy to to understand name